File: Submissions/#0366_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n;
  cin >> n;

  string hate = "I hate ";
  string love = "I love ";
  string answer = "";

  for (int i = 0; i < n; i++) {

    if (i % 2 == 0) {
      answer += hate;
    } else {
      answer += love;
    }

    if (i == n - 1) {
      answer += "it";
    } else {
      answer += "that ";
    }

  }

  cout << answer << endl;

}

File: Submissions/#0366_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    sc.close();

    String hate = "I hate ";
    String love = "I love ";
    String answer = "";

    for (int i = 0; i < n; i++) {

      if (i % 2 == 0) {
        answer += hate;
      } else {
        answer += love;
      }

      if (i == n - 1) {
        answer += "it";
      } else {
        answer += "that ";
      }

    }

    System.out.println(answer);

  }

}